src/components/Banner.jsx

```diff
@@ -13,18 +13,22 @@ class Banner extends React.Component {
     };
   }
 
+  componentDidMount(){
+
+  }
+
   renderLogos = allLogos => (
     <div className="card-container">
       <Card className="card">
-        <Card.Content className="card-content">
```

Comment:
Is it really necessary to put it in a div instead of Card.Content? The latter would be nice for the sake of using it how it's explained in the semantic ui docs.

Reply:
When using Card.Content, the whole thing breaks...